pages/_document.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
This is all copied from https://github.com/zeit/next.js/blob/canary/examples/with-styled-components/pages/_document.js